packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/plugin-ledger-connector-fabric.ts

```diff
@@ -1102,6 +1105,23 @@ export class PluginLedgerConnectorFabric
       let transactionId = "";
       switch (invocationType) {
         case FabricContractInvocationType.Call: {
+          if (fnName === "getLastBlock") {
```

Comment:
@barnapa If we continue down this path the function here will grow to infinity in term of LoC.
I recommend creating a `get-last-block-v1` endpoint instead which then can have a `getLastBlockV1()` method here in the connector that holds nothing but the exact logic what you need.
It also reduces the number of if conditions in the code by 1 which is always a win in my book because it reduces complexity and increases readability.

Reply:
get-last-block-v1 endpoint in which plugin ? connector or persistence ?

Reply:
ok checking on that

Reply:
@petermetz For now deleted this functionality.